ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.cpp

```diff
@@ -71,6 +71,7 @@ ReadSymbolsFromMobileStyle::ReadSymbolsFromMobileStyle(QObject* parent /* = null
     m_models[index] = results;
 
     emit symbolResultsChanged();
+    updateSymbol(0, 0, 0, Qt::yellow, 40);
```

Comment:
This should be `QColor(Qt::yellow)` for readability and to avoid implicit conversion.

Reply:
alright, should be updated now. thanks